doc/api/stream.md

```diff
@@ -1797,6 +1797,10 @@ class SourceWrapper extends Readable {
 *Note*: The `readable.push()` method is intended be called only by Readable
 Implementers, and only from within the `readable._read()` method.
 
+*Note*: For streams not operating in object mode, if the `chunk` parameter
+of `readable.push()` is `undefined`, it will be treated as empty string or
+buffer.
```

Comment:
Perhaps just remove the `*Note*:` part, it's really not necessary

Reply:
because I don't know where it should be placed.

Reply:
@jasnell fixed